src/create-test/mocha.js

```diff
@@ -1,8 +1,8 @@
 export default function createMochaTest({actions, imports, reducer, equalityFunction}) {
   const its = actions.map((action, index) => {
     return `  it('${action.action.type} (action index ${index}) should correctly update state', function() {
-  var action = actions[${index}];
-  var result = reducer(active.prevState, action.action);
+  var action = actions[${index}];a
```

Comment:
thanks! there's an errant a here a the end of the line, but if you remove that, i can merge and publish a new version